src/app/file-manager/services/selection.service.ts

```diff
@@ -31,7 +31,9 @@ export class SelectionService {
     }
 
     nothingSelected(): boolean {
-        return this.selection.selected.length <= 0;
+        return this.selection.selected.length === 1
+            && this.selection.selected[0].id === 'workspaces'
+            || this.selection.selected.length <= 0 ? true : false;
```

Comment:
Why `? true : false`?  Isn't that the same as returning the boolean result of the expression?